### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by Mlg29**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I need help writing tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/offline-pay-with-crypto-widget/src/components/Modal/index.js [hunk 1]
```diff
@@ -1,3 +1,27 @@
+<<<<<<< HEAD
```
Comment (uchibeke) [inline packages/offline-pay-with-crypto-widget/src/components/Modal/index.js:1]: there is a merge issue here

File: packages/offline-pay-with-crypto-widget/src/components/Modal/index.js [hunk 2]
```diff
@@ -14,6 +38,7 @@ import cryptoIcon from 'base64-cryptocurrency-icons'
 import { ContractKitProvider } from '@celo-tools/use-contractkit'
 import '@celo-tools/use-contractkit/lib/styles.css'
 import { useContractKit } from '@celo-tools/use-contractkit'
+>>>>>>> d0b77439c9d30cb6bf7a079100ce9e81af0f72d3
```
Comment (uchibeke) [inline packages/offline-pay-with-crypto-widget/src/components/Modal/index.js:41]: also here. please run the cypress tests from package.json

File: packages/offline-pay-with-crypto-widget/src/components/Modal/index.js [hunk 3]
```diff
@@ -58,22 +83,101 @@ const ModalComponent = ({
       return true
     }
 
+<<<<<<< HEAD
```
Comment (uchibeke) [inline packages/offline-pay-with-crypto-widget/src/components/Modal/index.js:86]: merge issue

File: packages/offline-pay-with-crypto-widget/src/components/Modal/index.js [hunk 4]
```diff
@@ -134,6 +238,54 @@ const ModalComponent = ({
 
   return (
     <motion.div
+<<<<<<< HEAD
```
Comment (uchibeke) [inline packages/offline-pay-with-crypto-widget/src/components/Modal/index.js:241]: merge issue

**Review by uchibeke — CHANGES_REQUESTED** [review]
please fix merge issues and run cypress tests

File: packages/offline-pay-with-crypto-widget/src/components/Modal/index.js (hunk 2 above)
Comment (Mlg29) [reply packages/offline-pay-with-crypto-widget/src/components/Modal/index.js:41]: how do i run cypress ?

**Pull request closed without merging**